client/src/index.js

```diff
@@ -4,13 +4,15 @@ import "./index.css";
 import App from "./components/App";
 import VerticalTabs from "./components/Dashboard/dashboard"
 import NavBar from "./components/navBar/navBar"
+import Flights from "./components/Dashboard/flights/flights"
```

Comment:
this doesn't want to be in here, it should only be put in here for checking it's working. This will sit in the dashboard component

Reply:
Yes, changed to be rendered from dashboard